web-client/src/views/TrialSessionWorkingCopy/CaseListRowTrialSession.jsx

```diff
@@ -58,12 +61,27 @@ const getCaseRow = ({
               });
             }}
           >
-            <option value="">-Trial Status-</option>
-            {trialStatusOptions.map(({ key, value }) => (
-              <option key={key} value={key}>
-                {value}
-              </option>
-            ))}
+            <option value="">-{unassignedLabel}-</option>
+            {Object.keys(trialStatusOptions).map(key => {
+              if (updatedTrialSessionTypesEnabled) {
+                if (
+                  !trialStatusOptions[key].deprecated ||
+                  trialSessionWorkingCopyStatus === key
```

Comment:
Can this `trialSessionWorkingCopyStatus` be reused on line 55? And to clarify, this is the case trial status on the working copy, right?

Reply:
Correct on both counts!